drivers/LKCoreFs/include/LKFile.h

```diff
@@ -0,0 +1,31 @@
+#ifndef _LEKA_OS_DRIVER_LKFILE_H_
+#define _LEKA_OS_DRIVER_LKFILE_H_
+
+#include <cstdio>
+
+#include "interface/LKFile.h"
+
+namespace leka {
+
+class LKFile : public interface::LKFile
```

Comment:
> why class and not struct

Oups right, should stick to one, struct makes sense

> maybe we could move it CoreFileSystem to have

I fear this would become redundant and long to type; `leka::File` looks explicit enough ?